packages/@azure/servicebus/data-plane/test/streamingReceiverSessions.spec.ts

```diff
@@ -0,0 +1,1084 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import chai from "chai";
+const should = chai.should();
+import chaiAsPromised from "chai-as-promised";
+import dotenv from "dotenv";
+dotenv.config();
+chai.use(chaiAsPromised);
+import {
+  Namespace,
+  QueueClient,
+  MessageSession,
+  ServiceBusMessage,
+  TopicClient,
+  SubscriptionClient,
+  delay
+} from "../lib";
+
+import { DispositionType } from "../lib/serviceBusMessage";
+
+import { testMessagesWithSessions, testSessionId } from "./testUtils";
+
+async function testPeekMsgsLength(
+  client: QueueClient | SubscriptionClient | MessageSession,
+  expectedPeekLength: number
+): Promise<void> {
+  const peekedMsgs = await client.peek(expectedPeekLength + 1);
+  should.equal(
+    peekedMsgs.length,
+    expectedPeekLength,
+    "Unexpected number of msgs found when peeking"
+  );
+}
+
+let namespace: Namespace;
+
+// let partitionedQueueClient: QueueClient;
+// let partitionedDeadletterQueueClient: QueueClient;
```

Comment:
We dont need these as they dont support sessions correct?
Let's remove them.
Same for all the other commented code